index.js

```diff
@@ -80,10 +80,16 @@ function getVersion(repoPath) {
   return pkg.version;
 }
 
-function generateNextVersion(currentVersion, level) {
-  const version = semver.inc(currentVersion, level)
+function generateNextVersion(currentVersion, level, preId) {
+  const version = semver.inc(currentVersion, level, preId);
```

Comment:
I don't see this argument in the documentation [here](https://github.com/npm/node-semver#functions). Did the author forget to document it?

They do say "All methods and classes take a final loose boolean argument that, if true, will be more forgiving about not-quite-valid semver strings. The resulting output will always be 100% strict, of course." but I'm not clear about what is going on here.

Reply:
The author documents more on that usage here, specifically with that argument being used for preids: https://github.com/npm/node-semver#prerelease-identifiers